src/main/java/com/oath/halodb/FileUtils.java

```diff
@@ -30,6 +34,21 @@ static void createDirectoryIfNotExists(File directory) throws IOException {
         }
     }
 
+    static void deleteDirectory(File dir) throws IOException {
+        File[] files = dir.listFiles();
+        if (files != null) {
+            for (File file : files) {
+                if (file.isDirectory()) {
+                    FileUtils.deleteDirectory(file);
```

Comment:
```deleteDirectory(file);``` . ```FileUtils``` is not needed. It will make code ```recursion``` legible

Reply:
In fact calling FileUtils.deleteDirectory(file) and deleteDirectory(file) should be exactly same except the format. 

What do you mean recursion legible?